.github/workflows/llvmlite_osx-64_wheel_builder.yml

```diff
@@ -66,10 +66,11 @@ jobs:
         run: |
           if [ "${{ inputs.llvmdev_run_id }}" != "" ]; then
               CHAN="file://${{ github.workspace }}/llvmdev_conda_packages"
+              conda install -c "$CHAN" llvmdev=${{ env.FALLBACK_LLVMDEV_VERSION }}
```

Comment:
Shouldn't this also use `-c defaults` for consistency?

Reply:
fixed in 458a614848e6e19a342ec701ae087d478b5dfa9b